USER_GUIDE.md

```diff
@@ -0,0 +1,123 @@
+- [Setup](#setup)
+- [Sample code](#sample-code)
+- [AWS Managed Opensearch with Sigv4](#aws-managed-opensearch-with-sigv4)
+
+## Setup
+
+To add the client to your project, install it using [RubyGem](https://rubygems.org/)
+
+`gem install opensearch-ruby`
+
+or, install it from a source code checkout:
+```bash
+git clone https://github.com/opensearch-project/opensearch-ruby.git
+cd opensearch-ruby/opensearch
+gem build opensearch.gemspec
+gem install opensearch-ruby-<version>.gem
+```
+
+Import the client as a module:
```

Comment:
Technically correct, but there's no other way to import something in Ruby, so I would just say "import the client".